src/datetimepicker.ios.js

```diff
@@ -69,6 +69,7 @@ export default class Picker extends React.Component<IOSNativeProps> {
       mode,
       minuteInterval,
       timeZoneOffsetInMinutes,
+      iOsPickerStyle = {},
```

Comment:
I'd vote for this prop to be named just `style`. There are several other props such as `timeZoneOffsetInMinutes` that only work on one platform, and we never use any ios/android suffix/prefix to distinguish those.

Reply:
Do you suggest to use same styles on outer container and also DatePicker?

Reply:
Style is already used here - https://github.com/react-native-community/react-native-datetimepicker/blob/e6d2980230b1e27fa8aed989ccbb79897e6ef5eb/src/datetimepicker.ios.js#L80

Reply:
huh I didn't know that! (it's not documented). That brings me to a question - why is `RNDateTimePicker` wrapped in a View? Would it make sense to remove that wrapping view?

Reply:
I can remove it, I've tested it and it works without the external View.
I just wonder whether there will be breaking change in case someone had a hack to get around inability to change height of the datepicker.

Reply:
Alright added it to example to `App.js` and has documentation in `Readme.md`. I've kept it as **iOS only** due to the fact that you can't really style Android pickers this way.

Reply:
> I can remove it, I've tested it and it works without the external View.

in that case I'd remove it, and if folks need to wrap it, they can do it themselves

I checked https://github.com/facebook/react-native/blame/master/Libraries/Components/DatePicker/DatePickerIOS.ios.js and from the blame, it doesn't seem like there was specific reason for adding the wrapping view

Reply:
Ok done